internal/converter/converter.go

```diff
@@ -180,6 +182,38 @@ func ConvertWithOptions(req *pluginpb.CodeGeneratorRequest, opts options.Options
 	}, nil
 }
 
+func mergeTags(tags []*base.Tag) []*base.Tag {
+
+	if len(tags) == 0 {
+		return tags
+	}
+
+	res := make([]*base.Tag, 0, len(tags))
+	found := make(map[string]*base.Tag)
```

Comment:
Since the values stored in `found` are never used you can make this a `map[string]struct{}{}` instead which is slightly more efficient (which probably doesn't matter in this case) but also with `struct{}` as the value of the map no one will mistake it as something that's going to get used other